assets/src/edit-story/panels/size.js

```diff
@@ -61,20 +65,9 @@ function SizePanel( { selectedElements, onSetProperties } ) {
 						width: typeof newHeight === 'number' && lockRatio ? newHeight * ratio : width,
 					} );
 				} }
-				postfix={ _x( 'px', 'pixels, the measurement of size', 'amp' ) }
-				disabled={ isFullbleed }
```

Comment:
You can't remove the `isFullBleed` functionality. If an element is taking up the entire page by that setting, changing width and height here has no meaning.

Reply:
isFullBleed not removed. It's still there